docs/DeveloperGuide.md

```diff
@@ -112,6 +112,11 @@ Named Arguments:
 --ignore-standalone-config, -i
                      A flag to ignore the standalone config file in
                      the repo.
+--timezone ZONE_ID[±hh[mm]], -t ZONE_ID[±hh[mm]]
+                     The timezone to use for the generated report. 
+                     One kind of valid timezones is relative to UTC. 
+                     E.g. UTC, UTC+08, UTC-1030.
```

Comment:
@damithc Just to confirm, did you meant to give multiple example usages of `UTC` timezone, or to give multiple timezone examples that is accepted by us?

Reply:
Multiple examples of UTC is enough 👍